src/eko/kernels/singlet.py

```diff
@@ -359,7 +359,7 @@ def eko_iterate(gamma_singlet, a1, a0, beta_vec, order, ev_op_iterations):
 
 
 @nb.njit(cache=True)
-def r_vec(gamma_singlet, beta, ev_op_max_order, order, is_exact):
+def r_vec(gamma_singlet, beta, ev_op_max_order, order, is_exact, dim=2):
```

Comment:
Not sure that Numba appreciates default for arguments:
https://numba.pydata.org/numba-doc/dev/reference/types.html?highlight=optional#optional-types

Just drop the default, and plug it where it was originally called.